R/col_types.R

```diff
@@ -199,12 +202,13 @@ format.col_spec <- function(x, n = Inf, condense = NULL, colour = crayon::has_co
   delim <- x$delim
 
   if (!is.null(delim) && nzchar(delim)) {
-    delim <- paste0('.delim = ', double_quote(delim), '')
+    delim <- paste0(".delim = ", double_quote(delim), "")
```

Comment:
This look suspicious to me. Given the presence of a call to `double_quote()`, I suspect there could be a very good reason for using single quotes. You need to scrutinize this.

Reply:
Yeah, I think this does cause test failure, BTW.

Reply:
I did go through each file that changed, line by line, and tried to scrutinize these changes and there were some changes that I ignored. But as you can see there were a lot of files. I've also never come across style changes that would break tests so that was not something that was on my mind but it will be in the future.

Reply:
Oh, I am sure you did look through this all carefully! Sorry if my "you need to scrutinize this" sounded like a general comment. I wrote it when I first started to look at this and thought we'd actually be merging this PR. And I wanted you to revisit this spot in particular. It is actually a case where the tests worked and would have lead to further investigation anyway.

But I now propose you note our decision to walk away from this in #407, strike the `use_tidy_style()` item out, and close this unmarked.

Reply:
Ah, gotcha! I'll make note of this in #407 and close this PR.